packages/components/src/MessageBar/MessageBar.tsx

```diff
@@ -247,10 +266,9 @@ export const MessageBar = styled(MessageBarLayout).attrs(
   })
 )`
   ${simpleLayoutCSS}
-
   align-items: center;
-  background: ${({ intent, theme: { colors } }) =>
-    intent === 'critical' ? colors.criticalAccent : colors.neutralAccent};
+  ${({ intent, theme: { colors } }) =>
+    intent ? backgroundColor : `background: ${colors.informAccent};`}
   border-radius: ${({ theme: { radii } }) => radii.medium};
```

Comment:
Since you're working here anyway it'd be great to fix another problem I've seen:

```suggestion
  border-radius: ${({ theme: { radii } }) => radii.medium};
  color: ${({ theme }) => theme.color.text };
```

Currently color is inherited instead of explicit so when themes are inverted things look sad.

Reply:
![image](https://user-images.githubusercontent.com/170681/108073584-12667880-701d-11eb-9c25-3e4f0b625f79.png)